egret/model_library/transmission/branch.py

```diff
@@ -801,6 +865,8 @@ def declare_ineq_p_branch_thermal_bounds(model, index_set,
     m = model
     con_set = decl.declare_set('_con_ineq_p_branch_thermal_lbub',
                                model=model, index_set=index_set)
+    pos_slacks = hasattr(model, 'pf_slack_pos')
+    neg_slacks = hasattr(model, 'pf_slack_neg')
```

Comment:
Same comment as above about using a function argument to indicate whether or not slacks should be used.

Reply:
Changed to use a function argument similar to above.